editor/scripts/bundle.py

```diff
@@ -356,6 +357,20 @@ def remove_platform_files_from_archive(platform, jar):
                 continue
             # anything else should be removed
             files_to_remove.append(file)
+        # find files to remove `.pdb` from _updack/*
+        if file.startswith("_unpack"):
+            # don't touch anything for the current platform
+            if file.startswith(_unpack_platform):
+                continue
+            # keep shared files
+            if file.startswith("_unpack/shared"):
+                continue
+            # keep _defold folder
+            if file.startswith("_unpack/_defold"):
+                continue
+            # anything else should be removed
+            files_to_remove.append(file)
```

Comment:
We use unpack folder only for the editor: 
- build and run
- executables for building process and luajit
- libs e.g. particles etc

This folder contains everything we need to be able to run the project or build it, not bundle. That means we can remove all the files for all the other platforms and do not unpack it all the time.